### PR TITLE
better error msg

### PR DESCRIPTION
**Pull request by utle**
Add better error message.

### CODE REVIEW

**Comment by utle** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.spnego_fat,com.ibm.ws.security.openidconnect.server_fat.spnego,com.ibm.ws.security.openidconnect.client_fat.spnego

File: dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops [hunk 1]
```diff
@@ -102,9 +102,9 @@ SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR=CWWKS4319E: Can not get the content
...
 SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR.useraction=Review the custom error page content type to make sure it is valid.
 
-SPNEGO_CAN_NOT_VALIDATE_TOKEN=CWWKS4320E: The SPNEGO or Kerberos token included in the HttpServletRequest can not be validated {0}
-SPNEGO_CAN_NOT_VALIDATE_TOKEN.explanation=The SPNEGO or Kerberos token used to authenticate the request is not valid.
-SPNEGO_CAN_NOT_VALIDATE_TOKEN.useraction=If no additional error message is displayed, please enable JGSS and KRB5 debugging. Please see the JDK documentation for how to enable JGSS and KRB5 debugging. Verify the SPNEGO configuration and make sure the client, SPN and Kerberos keytab files use the same encryption type that is specified in the Kerberos configuration file.
+SPNEGO_CAN_NOT_ESTABLISH_GSSCONTEXT_WITH_VALIDATE_TOKEN=CWWKS4320E: Can not establish the GSSContext with a valid SPNEGO or Kerberos token included in the HttpServletRequest {0} 
+SPNEGO_CAN_NOT_ESTABLISH_GSSCONTEXT_WITH_VALIDATE_TOKEN.explanation=The SPNEGO or Kerberos token used to authenticate the request is valid but we can not establish the GSSContext.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops]: change
valid but we can not establish the GSSContext.
to
valid, but the GSSContext cannot be established.

File: dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops [hunk 2]
```diff
@@ -102,9 +102,9 @@ SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR=CWWKS4319E: Can not get the content
...
+SPNEGO_CAN_NOT_ESTABLISH_GSSCONTEXT_WITH_VALIDATE_TOKEN.useraction=If no additional error message is displayed, please enable JGSS and KRB5 debugging. Please see the JDK documentation for how to enable JGSS and KRB5 debugging. Verify the SPNEGO configuration and make sure the client, SPN and Kerberos keytab files use the same encryption type that is specified in the Kerberos configuration file.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops]: change
If no additional error message is displayed, please enable JGSS and KRB5 debugging.
to
If no additional error message is displayed, enable JGSS and KRB5 debugging.

change
Please see the JDK documentation for how to enable JGSS and KRB5 debugging.
to
Refer to the JDK documentation for instructions on enabling JGSS and KRB5 debugging.

change 
client, SPN and Kerberos
to
client, SPN, and Kerberos

File: dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops [hunk 3]
```diff
@@ -102,9 +102,9 @@ SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR=CWWKS4319E: Can not get the content
 SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR.explanation=The custom error page cannot be loaded due to the exception when getting the content type.
 SPNEGO_CUSTOM_ERROR_PAGE_CONTENT_TYPE_ERROR.useraction=Review the custom error page content type to make sure it is valid.
 
-SPNEGO_CAN_NOT_VALIDATE_TOKEN=CWWKS4320E: The SPNEGO or Kerberos token included in the HttpServletRequest can not be validated {0}
-SPNEGO_CAN_NOT_VALIDATE_TOKEN.explanation=The SPNEGO or Kerberos token used to authenticate the request is not valid.
-SPNEGO_CAN_NOT_VALIDATE_TOKEN.useraction=If no additional error message is displayed, please enable JGSS and KRB5 debugging. Please see the JDK documentation for how to enable JGSS and KRB5 debugging. Verify the SPNEGO configuration and make sure the client, SPN and Kerberos keytab files use the same encryption type that is specified in the Kerberos configuration file.
+SPNEGO_CAN_NOT_ESTABLISH_GSSCONTEXT_WITH_VALIDATE_TOKEN=CWWKS4320E: Can not establish the GSSContext with a valid SPNEGO or Kerberos token included in the HttpServletRequest {0}
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops]: change
Can not establish the GSSContext with a valid SPNEGO or Kerberos token included in the HttpServletRequest {0}
to
Cannot establish the GSSContext with a valid SPNEGO or Kerberos token included in the HttpServletRequest {0}

File: dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops (hunk 1 above)
Comment (utle) [reply dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops]: I made some minor changes such as remove class and interface.

**Comment by utle** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.spnego_fat,com.ibm.ws.security.openidconnect.server_fat.spnego,com.ibm.ws.security.openidconnect.client_fat.spnego

**Comment by utle** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.spnego_fat,com.ibm.ws.security.openidconnect.server_fat.spnego,com.ibm.ws.security.openidconnect.client_fat.spnego

**Pull request merged**